### ISSUE TITLE
About fine-tuned inception model file

### ISSUE THREAD

**Issue by bravotty** [kkanshul/finegan#7]
Hi! When i test your fine-tuned model file, it happens some errors:
`Error(s) in loading state_dict for Inception3:
	Missing key(s) in state_dict: "AuxLogits.fc.weight", "AuxLogits.fc.bias", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "fc_new.weight", "fc_new.bias", "AuxLogits.fc_new.weight", "AuxLogits.fc_new.bias". `
Could u mind send a fine-tuned inception model file(inceptionv2.py) for me, thx!!!

Other Question~, how to fine-tuned the inception model on bird datasets?

**Comment by utkarshojha** [comment]
Hi! Thanks for pointing that out. I've added the new inception model here (https://github.com/kkanshul/finegan/blob/master/code/inception.py). Right now it is for birds, and if you need it for some other dataset, just change the last fully connected layer dimension (200 for cars, 120 for dogs). 

As for the fine tuning of inception net, I just initialize the layers (except the last fully connected layer) with the inception network trained on ImageNet dataset. I then further train the network on the complete CUB dataset. I create a 80-20 split of CUB and use the 20 split as validation set to determine when to stop training the model.

Thanks ~

**Comment by bravotty** [comment]
thx!
